lib/msal-browser/src/app/ClientApplication.ts

```diff
@@ -1036,7 +1036,8 @@ export abstract class ClientApplication {
                 clientCapabilities: this.config.auth.clientCapabilities
             },
             systemOptions: {
-                tokenRenewalOffsetSeconds: this.config.system.tokenRenewalOffsetSeconds
+                tokenRenewalOffsetSeconds: this.config.system.tokenRenewalOffsetSeconds,
+                preventCorsPreflight: true
```

Comment:
I'm not sure I agree with exposing this to applications to control, given the server has asked us to prevent preflight requests from ever happening.

Reply:
Specifically for msal-browser correct? This isn't being exposed to applications, only to the platform libs.

Reply:
for msal-node, we should be sending items in headers when we can.

Reply:
Ah, I see.